powershell/VstsTaskSdk/ServerOMFunctions.ps1

```diff
@@ -598,6 +598,12 @@ function Get-OMType {
                     $dllPaths += [System.IO.Path]::Combine($OMDirectory, "$namespace.dll")
                 }
             }
+            if ($namespace -like 'Microsoft.TeamFoundation.Build.WebApi.*' -or
```

Comment:
I think this should be moved into the extended client block

Reply:
actually i'm not sure which block it should be moved into, but I think it should be moved into one of them. which one was the use case?

Reply:
I think it was from the WebApi, though its a little bit unclear since the project has since moved to a TypeScript implementation. Not 100% sure I understand the rationale for moving it into the blocks anyways though - couldn't this cause an issue with either?